pkg/webui/console/store/reducers/user.js

```diff
@@ -51,11 +51,10 @@ const user = function(state = defaultState, { type, payload }) {
         ...state,
         rights: payload,
       }
-    case LOGOUT_SUCCESS:
+    case LOGOUT_FAILURE:
       return {
         ...state,
-        user: undefined,
-        authInfo: undefined,
+        error: payload,
```

Comment:
`error` is not selected anywhere. no error is shown on failed logout

Reply:
Indeed. I've included a very basic passing of the error now as interim solution. This will be fleshed out properly [with the next PR](#2318).